test/integration.test.ts

```diff
@@ -156,31 +173,57 @@ describe('Starting cardano-wallet (and its node)', () => {
     await cleanupLauncher();
   });
 
-  it('Accepts a WriteStream, and pipes the child process stdout and stderr streams', () =>
-    tmp.withFile(async (logFile: tmp.FileResult) => {
-      const childProcessLogWriteStream = fs.createWriteStream(logFile.path, {
-        fd: logFile.fd,
-      });
-      const launcher = new Launcher({
-        stateDir: (
-          await tmp.dir({
-            unsafeCleanup: true,
-            prefix: 'launcher-integration-test-2',
-          })
-        ).path,
-        networkName: 'self',
-        nodeConfig: {
-          kind: 'jormungandr',
-          configurationDir: path.resolve(__dirname, 'data', 'jormungandr'),
-          network: jormungandr.networks.self,
-        },
-        childProcessLogWriteStream,
-      });
-      await launcher.start();
-      const logFileStats = await stat(logFile.path);
-      expect(logFileStats.size).toBeGreaterThan(0);
-      await launcher.stop();
-    }));
+  it('Accepts a WriteStream, and pipes the child process stdout and stderr streams', async () => {
+    const walletLogFile = await tmp.file();
+    const nodeLogFile = await tmp.file();
+    const launcher = new Launcher({
+      stateDir: (
+        await tmp.dir({
+          unsafeCleanup: true,
+          prefix: 'launcher-integration-test-',
+        })
+      ).path,
+      networkName: 'self',
+      nodeConfig: {
+        kind: 'jormungandr',
+        configurationDir: path.resolve(__dirname, 'data', 'jormungandr'),
+        network: jormungandr.networks.self,
+      },
+      childProcessLogWriteStreams: {
```

Comment:
@nikolaglumac :point_up:

Reply:
@rhyslbw we can still pass in the same stream for both node and wallet to achieve single file logging scenario right?

Reply:
That would be a good test case to add.

Reply:
done